level/level.go

```diff
@@ -100,6 +124,32 @@ func allowed(allowed level) Option {
 	return func(l *logger) { l.allowed = allowed }
 }
 
+// Parse a string to its corresponding level value. Valid strings are "debug",
+// "info", "warn", and "error". Strings are normalized via strings.TrimSpace and
+// strings.ToLower.
+func Parse(level string) (Value, error) {
```

Comment:
If the only returned error is invalid level, then maybe the second argument could simply be a boolran?

Reply:
isn't more idiomatic to return an error? maybe in the future we might return different types of errors and we'll not need to introduce breaking changes in the API

Reply:
I agree, `error` is a bit more future proof and IMO not meaningfully different for callers.